src/base/bootstrap-overrides/_base.scss

```diff
@@ -45,8 +45,24 @@ a {
 }
 
 .dl-horizontal {
-	dt {
-		white-space: normal;
+	@media (min-width: 768px) {
```

Comment:
Bootstrap uses this media query a lot in it's code. It seems like he just copied it.